src/App.tsx

```diff
@@ -15,48 +17,115 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Length = 'length',
+  Alphabet = 'alphabet',
+  Default = '',
+}
+
+interface SortOptions {
+  sortField: SortType;
+  isReversed?: boolean;
+}
+
+function getPreparedGoods(
+  goods: string[],
+  { sortField, isReversed }: SortOptions,
```

Comment:
Why do you pass the second and third arguments as one object? 
What is the point in this?